new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -226,112 +256,380 @@ public static void initAll(@Namespaces(6) List<String> namespaces) {
     installMonitoringExporter();
 
     // create and verify WebLogic domain image using model in image with model files
-    String imageName = createAndVerifyDomainImage();
+    miiImage = createAndVerifyMiiImage();
 
-    // create and verify one cluster domain
+    // create and verify one cluster mii domain
     logger.info("Create domain and verify that it's running");
-    createAndVerifyDomain(imageName, domain1Uid);
+    createAndVerifyDomain(miiImage, domain1Uid, domain1Namespace, "FromModel", 1);
 
     // get a free node port for NGINX
-    nodeportshttp = getNextFreePort(30305, 30405);
-    int nodeportshttps = getNextFreePort(30443, 30543);
+    nodeportshttp1 = getNextFreePort(30305, 30405);
+    nodeportshttp2 = getNextFreePort(30305, 30405);
+    int nodeportshttps1 = getNextFreePort(30443, 30543);
 
     // install and verify NGINX
-    nginxHelmParams = installAndVerifyNginx(nginxNamespace, nodeportshttp, nodeportshttps);
+    nginxHelmParams1 = installAndVerifyNginx(nginxNamespace1, nodeportshttp1, nodeportshttps1);
     // create ingress for the domain
     logger.info("Creating ingress for domain {0} in namespace {1}", domain1Uid, domain1Namespace);
-    Map<String, Integer> clusterNameMsPortMap = new HashMap<>();
+    clusterNameMsPortMap = new HashMap<>();
     clusterNameMsPortMap.put(clusterName, managedServerPort);
-    ingressHostList =
-        createIngressForDomainAndVerify(domain1Uid, domain1Namespace, clusterNameMsPortMap);
+    ingressHost1List =
+        createIngressForDomainAndVerify(domain1Uid, domain1Namespace,clusterNameMsPortMap, false);
+
+    exporterUrl = String.format("http://%s:%s/wls-exporter/",K8S_NODEPORT_HOST,nodeportshttp1);
+
+    //create pv and pvc for monitoring
+    HashMap<String, String> labels = new HashMap<>();
+    labels.put("app", "monitoring");
+    labels.put("weblogic.domainUid", domain1Uid);
+    assertDoesNotThrow(() -> createPvAndPvc("prometheus", monitoringNS, labels));
+    assertDoesNotThrow(() -> createPvAndPvc("alertmanager",monitoringNS, labels));
+    assertDoesNotThrow(() -> createPvAndPvc("grafana", monitoringNS, labels));
 
   }
 
   /**
-   * Test covers the following use cases.
+   * Test covers end to end sample, provided in the Monitoring Exporter github project .
    * Create Prometheus, Grafana, Webhook, Coordinator
-   * create domain Model in Image with monitoring exporter
+   * create domain in Image with monitoring exporter
    * verify access to monitoring exporter WebLogic metrics via nginx
-   * check generated by monitoring exporter WebLogic metrics via Prometheus
+   * check generated by monitoring exporter WebLogic metrics via Prometheus, Grafana
+   * fire Alert using Webhook
```

Comment:
Please fix punctuation. It looks like this when javadoc is viewed.

Test covers end to end sample, provided in the Monitoring Exporter github project . Create Prometheus, Grafana, Webhook, Coordinator create domain in Image with monitoring exporter verify access to monitoring exporter WebLogic metrics via nginx check generated by monitoring exporter WebLogic metrics via Prometheus, Grafana fire Alert using Webhook change prometheus to add different domain to monitor

Reply:
done